src/app/util.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+
+export function randomUuid(): UUID {
+  const s = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    const r = Math.random() * 16 | 0; const
+      v = c == 'x' ? r : (r & 0x3 | 0x8);
+    return v.toString(16);
+  });
+  return new UUID(s);
+}
+
+export class UUID {
```

Comment:
Should be `Uuid` to be camelCase compliant (like `randomUuid()`)

Reply:
It's a type not a name - java has `UUID` not `Uuid`. Same for [python](https://docs.python.org/3/library/uuid.html#uuid.UUID)